src/meta/src/hummock/compaction/level_selector.rs

```diff
@@ -616,6 +617,50 @@ pub fn default_level_selector() -> Box<dyn LevelSelector> {
     Box::<DynamicLevelSelector>::default()
 }
 
+#[derive(Default)]
+pub struct EmergencySelector {}
```

Comment:
Please move to an independent file, this file is enough large....

Reply:
ok, I'm going to separate the implementation of the different selectors in a separate pr and put it in a independent file, like the picker

Reply:
https://github.com/risingwavelabs/risingwave/pull/12392